src/python/dxpy/scripts/dx.py

```diff
@@ -4965,7 +4971,7 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
 # wait
 #####################################
 parser_wait = subparsers.add_parser('wait', help='Wait for data object(s) to close or job(s) to finish',
-                                    description='Polls the state of specified data object(s) or job(s) until they are all in the desired state.  Waits until the "closed" state for a data object, and for any terminal state for a job ("terminated", "failed", or "done").  Exits with a non-zero code if a job reaches a terminal state that is not "done".',
+                                    description='Polls the state of specified data object(s) or job(s) until they are all in the desired state.  Waits until the "closed" state for a data object, and for any terminal state for a job ("terminated", "failed", or "done").  Exits with a non-zero code if a job reaches a terminal state that is not "done".  Can also provide a local file containing a list of data object(s) or job(s), one per line.',
                                     prog='dx wait',
                                     parents=[env_args])
 path_action = parser_wait.add_argument('path', help='Path to a data object or job ID to wait for', nargs='+')
```

Comment:
Please, add "or a local file.." here since most people will just read (use as a reference) just the arguments. Thanks!